Source/UnitTests/UnitTests.vcxproj

```diff
@@ -56,7 +65,9 @@
     <ClCompile Include="$(ExternalsDir)gtest\src\gtest_main.cc" />
     <!--Lump all of the tests (and supporting code) into one binary-->
     <ClCompile Include="*.cpp" />
-    <ClCompile Include="*\*.cpp" />
+    <ClCompile Include="*\*.cpp">
```

Comment:
Not quite sure what's going on here/why some subset is excluded. 
In any case, does gtest support running cross-compiled builds in some way? This project may be mostly useless (at least for CI) on arm64 if not.

Reply:
The only problematic file is x64EmitterTest. But for now this part of the PR is fine.